src/services/study/sensitivity-analysis.js

```diff
@@ -108,7 +108,9 @@ export function getSensitivityAnalysisParameters(studyUuid) {
 
 export function fetchSensitivityAnalysisParameters(parameterUuid) {
     console.info('get sensitivity analysis parameters');
-    const url = `${process.env.REACT_APP_API_GATEWAY}/sensitivity-analysis/v1/parameters/${parameterUuid}`;
+    const url = `${
+        import.meta.env.VITE_API_GATEWAY
```

Comment:
You need to define this at the top of the file and use it here.
This way it can be reusable.
Search for "**import. meta. env. VITE_API_GATEWAY**" on other files as an  example